arcgis-ios-sdk-samples/Augmented reality/Navigate in AR/NavigateARNavigatorViewController.swift

```diff
@@ -249,7 +249,15 @@ extension NavigateARNavigatorViewController: AGSLocationChangeHandlerDelegate {
     func locationDataSource(_ locationDataSource: AGSLocationDataSource, locationDidChange location: AGSLocation) {
         startButtonItem.isEnabled = true
         // Send location updates to route tracker if actively navigating.
-        routeTracker?.trackLocation(location)
+        routeTracker?.trackLocation(location)  { [weak self] error in
```

Comment:
```suggestion
        routeTracker?.trackLocation(location) { [weak self] error in
```

Reply:
My bad here 😅 . What do you think for nilifying `locationChangeHandlerDelegate` vs `routeTracker` in this case?

Reply:
I don't really know which is the better approach.